src/encoded/schemas/changelogs/genetic_modification.md

```diff
@@ -3,6 +3,8 @@
 ### Minor changes since schema version 9
 
 * Added *disruption*, *inhibition*, and *knockout* as enums in *category* for use with CRISPR screen data.
+* Added *homologous recombination* as enum in *method*, to be used with *knockout category* and with the *purpose* of *repression*.
```

Comment:
probably should be *category* of *knockout* instead of *knockout category*

Reply:
Got it, I adjusted this and am resubmitting.